src/theme/DocVersionBanner/index.js

```diff
@@ -0,0 +1,39 @@
+// Why is this swizzled?
+//   To emit a custom banner for the Zeebe REST API Explorer, which has not been released yet.
+// Swizzled from version 2.4.1.
+
+// TODO: Delete this swizzled component as part of 8.5 release!
+
+import React from "react";
+import clsx from "clsx";
+import DocVersionBanner from "@theme-original/DocVersionBanner";
+import { useDocsVersion } from "@docusaurus/theme-common/internal";
+import { ThemeClassNames } from "@docusaurus/theme-common";
+
+export default function DocVersionBannerWrapper(props) {
+  const { pluginId } = useDocsVersion();
+  if (pluginId === "api-zeebe-docs") {
+    return <ZeebeRestApiBanner {...props} />;
+  }
+
+  return (
+    <>
+      <DocVersionBanner {...props} />
+    </>
+  );
+}
+
+function ZeebeRestApiBanner({ className }) {
+  return (
+    <div
+      className={clsx(
+        className,
+        ThemeClassNames.docs.docVersionBanner,
+        "alert alert--warning margin-bottom--md"
+      )}
+      role="alert"
+    >
+      <div>This is unreleased documentation for the Zeebe REST API.</div>
```

Comment:
@christinaausley @akeller this is where I'm emitting a message at the top of the Zeebe API Explorer to clarify that it is not yet released. It looks like this: 

![image](https://github.com/camunda/camunda-docs/assets/1627089/5479cb08-e466-4be4-bf69-ef3ba855b182)

The language is I think slightly confusing, because it's not just that the docs are unreleased, the entire API is (I think)....but it aligns with the messaging that shows up on vNext docs on the rest of the site: 

<img width="907" alt="image" src="https://github.com/camunda/camunda-docs/assets/1627089/d40e8f04-75e4-46ce-b15f-1c35d5cf8ec7">

Feel free to propose alternative text here!

cc @koevskinikola because the conversation in [this thread](https://github.com/camunda/camunda-docs/pull/3487#discussion_r1526492335) probably impacts this discussion.

Reply:
I wrote something similar in [the comment above](https://github.com/camunda/camunda-docs/pull/3487#discussion_r1531887349). I think it makes sense.